<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DelayedPulserWidget</class>
 <widget class="QWidget" name="DelayedPulserWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>20</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QESimpleShape" name="ledEnabled">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16</width>
       <height>20</height>
      </size>
     </property>
     <property name="shape">
      <enum>QSimpleShape::circle</enum>
     </property>
     <property name="colour0" stdset="0">
      <color>
       <red>20</red>
       <green>50</green>
       <blue>20</blue>
      </color>
     </property>
     <property name="colour1" stdset="0">
      <color>
       <red>5</red>
       <green>240</green>
       <blue>5</blue>
      </color>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):isEnabled</string>
     </property>
     <property name="variableSubstitutions" stdset="0">
      <string>ch=0</string>
     </property>
     <property name="displayAlarmStateOption" stdset="0">
      <enum>QE::Never</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QELabel" name="lblName">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):getName</string>
     </property>
     <property name="variableSubstitutions" stdset="0">
      <string>ch=0</string>
     </property>
     <property name="displayAlarmStateOption" stdset="0">
      <enum>QE::Always</enum>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>15</width>
       <height>17</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QENumericEdit" name="txtDelay">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>110</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>110</width>
       <height>20</height>
      </size>
     </property>
     <property name="displayAlarmStateOption" stdset="0">
      <enum>QE::Never</enum>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):setDelay</string>
     </property>
     <property name="variableSubstitutions" stdset="0">
      <string>ch=0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="separator">
      <enum>QE::NoSeparator</enum>
     </property>
     <property name="leadingZeros">
      <number>6</number>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000000.000000000000000</double>
     </property>
     <property name="addUnits">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QELabel" name="lblDelay">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>20</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):getDelay</string>
     </property>
     <property name="variableSubstitutions" stdset="0">
      <string>ch=0</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>15</width>
       <height>17</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QENumericEdit" name="txtWidth">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>20</height>
      </size>
     </property>
     <property name="displayAlarmStateOption" stdset="0">
      <enum>QE::Never</enum>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):setWidth</string>
     </property>
     <property name="variableSubstitutions" stdset="0">
      <string>ch=0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="leadingZeros">
      <number>6</number>
     </property>
     <property name="precision">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000000.000000000000000</double>
     </property>
     <property name="addUnits">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QELabel" name="lblWidth">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>20</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):getWidth</string>
     </property>
     <property name="variableSubstitutions" stdset="0">
      <string>ch=0</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>15</width>
       <height>17</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QEPushButton" name="btnOFF">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>19</height>
      </size>
     </property>
     <property name="text">
      <string>OFF</string>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):enable</string>
     </property>
     <property name="clickText">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QEPushButton" name="btnON">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>19</height>
      </size>
     </property>
     <property name="text">
      <string>ON</string>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):enable</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>15</width>
       <height>17</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QELineEdit" name="txtPrescaler">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>20</height>
      </size>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):setPrescaler</string>
     </property>
     <property name="displayAlarmStateOption" stdset="0">
      <enum>QE::Never</enum>
     </property>
     <property name="addUnits">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QELabel" name="lblPrescaler">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>20</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):getPrescaler</string>
     </property>
     <property name="displayAlarmStateOption" stdset="0">
      <enum>QE::Never</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QELabel</class>
   <extends>QLabel</extends>
   <header>QELabel.h</header>
  </customwidget>
  <customwidget>
   <class>QELineEdit</class>
   <extends>QLineEdit</extends>
   <header>QELineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>QENumericEdit</class>
   <extends>QWidget</extends>
   <header>QENumericEdit.h</header>
  </customwidget>
  <customwidget>
   <class>QEPushButton</class>
   <extends>QPushButton</extends>
   <header>QEPushButton.h</header>
  </customwidget>
  <customwidget>
   <class>QESimpleShape</class>
   <extends>QSimpleShape</extends>
   <header>QESimpleShape.h</header>
  </customwidget>
  <customwidget>
   <class>QSimpleShape</class>
   <extends>QWidget</extends>
   <header>QSimpleShape.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
